zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -302,25 +302,27 @@ impl<F: Field> Queries<F> {
             .chain(self.rw_counter_limbs.iter().rev())
             .cloned()
             .collect();
-        // most significant byte of id should be 0, so safe to overwrite it with packed
-        // tags.
-        limbs[0] = limbs[0].clone() + self.packed_tags() * (1u64 << 8).expr();
+        // most significant 9 bits of id are assumed be 0, so we can overwrite them with
+        // packed tags.
+        limbs[0] = limbs[0].clone() + self.packed_tags() * (1u64 << 7).expr();
```

Comment:
Perhaps add a hint 9 + 7 = 16 to be clear we are moving the value to the most significant 9 bits of the 16 bits?

Reply:
added comments in https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/523/commits/f8314b0ebe9246744344c56f02c89ab327ac83d3 thanks !